src/tests/ftest/erasurecode/ec_aggregation.yaml

```diff
@@ -0,0 +1,72 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+  test_clients:
+    - client-A
+    - client-B
+    - client-C
+timeout: 7200
+server_config:
+  engines_per_host: 2
+  name: daos_server
+  servers:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      bdev_class: nvme
+      bdev_list: ["aaaa:aa:aa.a"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem0"]
+      scm_mount: /mnt/daos0
+      log_mask: ERR
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      bdev_class: nvme
+      bdev_list: ["bbbb:bb:bb.b"]
+      scm_class: dcpm
+      scm_list: ["/dev/pmem1"]
+      scm_mount: /mnt/daos1
+      log_mask: ERR
+pool:
+    mode: 146
+    name: daos_server
+    scm_size: 200G
+    nvme_size: 500G
+    svcn: 1
+    control_method: dmg
+    pool_query_timeout: 30
+container:
+    type: POSIX
+    control_method: daos
```

Comment:
Suggestion: This can be done on different PR also... Try to enable checksum as a default on EC testing. I have noticed checksum and EC combination has issues... So adding to the test will improve more coverage.
    properties: cksum:crc64,cksum_size:16384,srv_cksum:on
You can also try to set the rf:1 value along with the properties string and see how things work.

If you don't pass oclass for container creation what is the default object class used for the container?

Reply:
That is a good suggestion but as you said things do not work as integrated so maybe we can move all the tests with checksum once that is stable.

I am using different RF properties but that is handled inside the code based on what EC type data will be written.

I think it's all depend upon RF and number of servers so if you have 6 engine and you have RF2 it will use EC_4P2G and RF:1 will have EC_4P1G.